application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -70,6 +70,22 @@ interface PlaceInterface {
   lon: Float
 }
 
+"Interface for rental vehicles, e.g. free floating vehicles and rental vehicle stations"
+interface VehicleRental {
```

Comment:
Create union for stations and free floating vehicles instead of interface.

Reply:
I chose `RentalPlace` as the union name because I think it's effective. But if you have a better idea I'll change it